lib/DDG/Spice/Recipes.pm

```diff
@@ -16,11 +16,11 @@ spice to => 'http://api.yummly.com/v1/api/recipes?q=$1&requirePictures=true&maxR
 handle query_lc => sub {
     return if grep {$req->query_lc eq $_} @stopwords;
 
-    if ($_ =~ s/recipes?//g) {
-        return trim($_);
+    my $ingredient_count = 0;
+    if ($_ =~ s/ ?recipes? ?//g) {
```

Comment:
@Mailkov @kwaping I would suggest we use `\brecipes\b`. The optional `\s?` means we could potentially match on a sub-string. The `\b` should be a little safer, and will also match `^|$`